lib/que/migrations/5/up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE que_jobs
+  ADD COLUMN que_version INTEGER DEFAULT 1;
+CREATE INDEX que_poll_idx_with_que_version ON que_jobs (que_version, queue, priority, run_at, id) WHERE (finished_at IS NULL AND expired_at IS NULL);
```

Comment:
How about we drop the versionless index, `que_poll_idx`? Seems like it'd be obsolete if we're going to include the version in queries.

Reply:
Yep, it is, but we want any old (version-ignorant) pollers to keep working between running this migration and deploying the app, so we'll remove the old index and rename this one in a subsequent migration.

Reply:
Ah, fair enough